### ISSUE TITLE
Generate and review validation reports: omeka

### ISSUE THREAD

**Issue by christinklez** [ucldc/rikolti#584]
- **Step 2: Generate reports** [status: in progress --> ready for review]
  - [x] Generate validation reports (run `validate_by_mapper DAG`): https://7a8067cb-3b99-477e-a883-7e311175a9b4.c3.us-west-2.airflow.amazonaws.com/dags/validate_by_mapper_type/grid?dag_run_id=manual__2023-10-31T22%3A57%3A23%2B00%3A00
  - [x] Add a comment (in this issue) to the output URLs
  - [x] Download the validation reports locally; prepend the collection registry number to the filename
  - [ ] Upload validation reports to the [Rikolti Data Validation](https://drive.google.com/drive/folders/1AgDPsz6oyDhsfgRuLyjomaXYgE1sPLNT) folder
- **Step 3: Review reports**  [status: ready for review --> review in progress]
  - [x] Review [sample collections validation reports](https://docs.google.com/spreadsheets/d/1XkBwi8jiuGgrWvQBqjkrbavSl13yyrQNyRyEb_CQ4z0/edit#gid=1143782640); leave notes in the spreadsheet
  - [x] Quickly (and not very deeply) scan through other validation reports (to identify if any new metadata fields surface)
  - [x] Synthesize notes at the [mapper level](https://docs.google.com/spreadsheets/d/1XkBwi8jiuGgrWvQBqjkrbavSl13yyrQNyRyEb_CQ4z0/edit#gid=1100474517)
  - [ ] Share an update (and surface any anomalies) during a weekly check-in (Mondays at 3:00pm)
- **Step 4: Have data validation issues? Address & resolve validation issues (synthesized mapper-level):**
  - [ ] Create a new issue (“Validation results: Rikolti mapper name“) and report/request CIC fixes & investigations. Assign to CIC.
  - [ ] Add a comment to link to the new “validation results” issue.
  - [ ] Add a “blocked/paused” label to this issue. Include a status note in this issue title (e,g., HOLD for CIC fixes; HOLD for code review; etc.) 
  - [ ] Move this issue back to the “To Do” pile.
- [Re-generate & re-review (repeat steps 2-3 above, as needed.]
- **Step 5: Have acceptable validation reports?**
  - [x] Move the valid (accepted) reports to the “VALID” folder in google drive.
  - [x] Update the [mapper progress spreadsheet](https://docs.google.com/spreadsheets/d/1XkBwi8jiuGgrWvQBqjkrbavSl13yyrQNyRyEb_CQ4z0/edit#gid=1100474517); indicate that the mapper is accepted

**Comment by christinklez** [comment]
Blocked by: https://github.com/ucldc/rikolti/issues/610

Once unblocked, return to the specific job & task, and re-run from the task that experienced the error. (No need to re-run prior successful tasks.)

**Comment by christinklez** [comment]
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/26724/validation/11-02-2023T17:04:17.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#26724/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/26725/validation/11-02-2023T17:04:20.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#26725/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27121/validation/11-02-2023T17:04:20.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27121/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27122/validation/11-02-2023T17:04:21.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27122/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27140/validation/11-02-2023T17:04:23.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27140/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27144/validation/11-02-2023T17:04:26.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27144/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27145/validation/11-02-2023T17:04:28.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27145/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27147/validation/11-02-2023T17:04:30.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27147/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27149/validation/11-02-2023T17:04:30.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27149/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27150/validation/11-02-2023T17:04:31.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27150/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27165/validation/11-02-2023T17:04:43.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27165/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27433/validation/11-02-2023T17:04:45.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27433/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27618/validation/11-02-2023T17:04:48.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27618/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27619/validation/11-02-2023T17:04:49.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27619/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27620/validation/11-02-2023T17:04:50.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27620/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27621/validation/11-02-2023T17:04:53.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27621/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27622/validation/11-02-2023T17:04:53.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27622/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27623/validation/11-02-2023T17:04:53.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27623/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27624/validation/11-02-2023T17:04:55.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27624/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27625/validation/11-02-2023T17:05:12.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27625/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27667/validation/11-02-2023T17:05:13.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27667/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27668/validation/11-02-2023T17:05:14.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27668/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27669/validation/11-02-2023T17:05:14.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27669/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27670/validation/11-02-2023T17:05:14.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27670/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27695/validation/11-02-2023T17:05:16.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27695/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27696/validation/11-02-2023T17:05:18.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27696/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27697/validation/11-02-2023T17:05:25.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27697/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27797/validation/11-02-2023T17:05:26.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27797/

[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27970/validation/11-02-2023T17:05:30.csv
[2023-11-02, 17:05:31 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27970/

**Comment by christinklez** [comment]
Validation results reported here: https://github.com/ucldc/rikolti/issues/625

**Comment by christinklez** [comment]
Fixes requested: #713 & #626

**Comment by christinklez** [comment]
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/26724/vernacular_metadata_2024-02-01T17:21:26/mapped_metadata_2024-02-01T17:27:01/validation_2024-02-01T17:31:03.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#26724/vernacular_metadata_2024-02-01T17:21:26/mapped_metadata_2024-02-01T17:27:01/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/26725/vernacular_metadata_2024-02-01T17:22:32/mapped_metadata_2024-02-01T17:27:17/validation_2024-02-01T17:31:05.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#26725/vernacular_metadata_2024-02-01T17:22:32/mapped_metadata_2024-02-01T17:27:17/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27121/vernacular_metadata_2024-02-01T17:22:41/mapped_metadata_2024-02-01T17:27:19/validation_2024-02-01T17:31:06.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27121/vernacular_metadata_2024-02-01T17:22:41/mapped_metadata_2024-02-01T17:27:19/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27122/vernacular_metadata_2024-02-01T17:22:43/mapped_metadata_2024-02-01T17:27:19/validation_2024-02-01T17:31:06.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27122/vernacular_metadata_2024-02-01T17:22:43/mapped_metadata_2024-02-01T17:27:19/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27140/vernacular_metadata_2024-02-01T17:22:44/mapped_metadata_2024-02-01T17:27:20/validation_2024-02-01T17:31:08.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27140/vernacular_metadata_2024-02-01T17:22:44/mapped_metadata_2024-02-01T17:27:20/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27144/vernacular_metadata_2024-02-01T17:22:48/mapped_metadata_2024-02-01T17:27:46/validation_2024-02-01T17:31:10.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27144/vernacular_metadata_2024-02-01T17:22:48/mapped_metadata_2024-02-01T17:27:46/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27145/vernacular_metadata_2024-02-01T17:22:54/mapped_metadata_2024-02-01T17:27:52/validation_2024-02-01T17:31:12.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27145/vernacular_metadata_2024-02-01T17:22:54/mapped_metadata_2024-02-01T17:27:52/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27147/vernacular_metadata_2024-02-01T17:22:58/mapped_metadata_2024-02-01T17:27:54/validation_2024-02-01T17:31:14.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27147/vernacular_metadata_2024-02-01T17:22:58/mapped_metadata_2024-02-01T17:27:54/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27149/vernacular_metadata_2024-02-01T17:23:02/mapped_metadata_2024-02-01T17:27:59/validation_2024-02-01T17:31:14.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27149/vernacular_metadata_2024-02-01T17:23:02/mapped_metadata_2024-02-01T17:27:59/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27150/vernacular_metadata_2024-02-01T17:23:03/mapped_metadata_2024-02-01T17:28:00/validation_2024-02-01T17:31:15.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27150/vernacular_metadata_2024-02-01T17:23:03/mapped_metadata_2024-02-01T17:28:00/data/
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27165/vernacular_metadata_2024-02-01T17:23:03/mapped_metadata_2024-02-01T17:28:04/validation_2024-02-01T17:31:24.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27165/vernacular_metadata_2024-02-01T17:23:03/mapped_metadata_2024-02-01T17:28:04/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27433/vernacular_metadata_2024-02-01T17:23:26/mapped_metadata_2024-02-01T17:28:48/validation_2024-02-01T17:31:26.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27433/vernacular_metadata_2024-02-01T17:23:26/mapped_metadata_2024-02-01T17:28:48/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27618/vernacular_metadata_2024-02-01T17:23:33/mapped_metadata_2024-02-01T17:28:50/validation_2024-02-01T17:31:28.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27618/vernacular_metadata_2024-02-01T17:23:33/mapped_metadata_2024-02-01T17:28:50/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27619/vernacular_metadata_2024-02-01T17:23:46/mapped_metadata_2024-02-01T17:28:53/validation_2024-02-01T17:31:29.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27619/vernacular_metadata_2024-02-01T17:23:46/mapped_metadata_2024-02-01T17:28:53/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27620/vernacular_metadata_2024-02-01T17:23:47/mapped_metadata_2024-02-01T17:28:54/validation_2024-02-01T17:31:29.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27620/vernacular_metadata_2024-02-01T17:23:47/mapped_metadata_2024-02-01T17:28:54/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27621/vernacular_metadata_2024-02-01T17:23:50/mapped_metadata_2024-02-01T17:29:07/validation_2024-02-01T17:31:32.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27621/vernacular_metadata_2024-02-01T17:23:50/mapped_metadata_2024-02-01T17:29:07/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27622/vernacular_metadata_2024-02-01T17:24:02/mapped_metadata_2024-02-01T17:29:11/validation_2024-02-01T17:31:32.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27622/vernacular_metadata_2024-02-01T17:24:02/mapped_metadata_2024-02-01T17:29:11/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27623/vernacular_metadata_2024-02-01T17:24:03/mapped_metadata_2024-02-01T17:29:13/validation_2024-02-01T17:31:32.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27623/vernacular_metadata_2024-02-01T17:24:03/mapped_metadata_2024-02-01T17:29:13/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27624/vernacular_metadata_2024-02-01T17:24:04/mapped_metadata_2024-02-01T17:29:15/validation_2024-02-01T17:31:34.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27624/vernacular_metadata_2024-02-01T17:24:04/mapped_metadata_2024-02-01T17:29:15/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27625/vernacular_metadata_2024-02-01T17:24:11/mapped_metadata_2024-02-01T17:29:20/validation_2024-02-01T17:31:51.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27625/vernacular_metadata_2024-02-01T17:24:11/mapped_metadata_2024-02-01T17:29:20/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27667/vernacular_metadata_2024-02-01T17:25:32/mapped_metadata_2024-02-01T17:30:12/validation_2024-02-01T17:31:51.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27667/vernacular_metadata_2024-02-01T17:25:32/mapped_metadata_2024-02-01T17:30:12/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27668/vernacular_metadata_2024-02-01T17:25:33/mapped_metadata_2024-02-01T17:30:13/validation_2024-02-01T17:31:51.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27668/vernacular_metadata_2024-02-01T17:25:33/mapped_metadata_2024-02-01T17:30:13/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27669/vernacular_metadata_2024-02-01T17:25:34/mapped_metadata_2024-02-01T17:30:16/validation_2024-02-01T17:31:52.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27669/vernacular_metadata_2024-02-01T17:25:34/mapped_metadata_2024-02-01T17:30:16/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27670/vernacular_metadata_2024-02-01T17:25:35/mapped_metadata_2024-02-01T17:30:19/validation_2024-02-01T17:31:52.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27670/vernacular_metadata_2024-02-01T17:25:35/mapped_metadata_2024-02-01T17:30:19/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27695/vernacular_metadata_2024-02-01T17:25:36/mapped_metadata_2024-02-01T17:30:20/validation_2024-02-01T17:31:54.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27695/vernacular_metadata_2024-02-01T17:25:36/mapped_metadata_2024-02-01T17:30:20/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27696/vernacular_metadata_2024-02-01T17:25:42/mapped_metadata_2024-02-01T17:30:21/validation_2024-02-01T17:31:56.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27696/vernacular_metadata_2024-02-01T17:25:42/mapped_metadata_2024-02-01T17:30:21/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27697/vernacular_metadata_2024-02-01T17:25:51/mapped_metadata_2024-02-01T17:30:24/validation_2024-02-01T17:32:03.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27697/vernacular_metadata_2024-02-01T17:25:51/mapped_metadata_2024-02-01T17:30:24/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27797/vernacular_metadata_2024-02-01T17:26:27/mapped_metadata_2024-02-01T17:30:31/validation_2024-02-01T17:32:04.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27797/vernacular_metadata_2024-02-01T17:26:27/mapped_metadata_2024-02-01T17:30:31/data/

[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27970/vernacular_metadata_2024-02-01T17:26:30/mapped_metadata_2024-02-01T17:30:32/validation_2024-02-01T17:32:09.csv
[2024-02-01, 17:32:09 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27970/vernacular_metadata_2024-02-01T17:26:30/mapped_metadata_2024-02-01T17:30:32/data/

**Comment by christinklez** [comment]
Reviewed, and new fixes requested in #750

**Comment by christinklez** [comment]
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/26724/vernacular_metadata_2024-02-02T23:40:36/mapped_metadata_2024-02-02T23:45:51/validation_2024-02-02T23:49:57.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#26724/vernacular_metadata_2024-02-02T23:40:36/mapped_metadata_2024-02-02T23:45:51/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27121/vernacular_metadata_2024-02-02T23:41:36/mapped_metadata_2024-02-02T23:46:09/validation_2024-02-02T23:49:58.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27121/vernacular_metadata_2024-02-02T23:41:36/mapped_metadata_2024-02-02T23:46:09/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27122/vernacular_metadata_2024-02-02T23:41:38/mapped_metadata_2024-02-02T23:46:09/validation_2024-02-02T23:49:59.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27122/vernacular_metadata_2024-02-02T23:41:38/mapped_metadata_2024-02-02T23:46:09/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27140/vernacular_metadata_2024-02-02T23:41:39/mapped_metadata_2024-02-02T23:46:09/validation_2024-02-02T23:50:00.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27140/vernacular_metadata_2024-02-02T23:41:39/mapped_metadata_2024-02-02T23:46:09/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27144/vernacular_metadata_2024-02-02T23:41:42/mapped_metadata_2024-02-02T23:46:37/validation_2024-02-02T23:50:03.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27144/vernacular_metadata_2024-02-02T23:41:42/mapped_metadata_2024-02-02T23:46:37/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27145/vernacular_metadata_2024-02-02T23:41:49/mapped_metadata_2024-02-02T23:46:42/validation_2024-02-02T23:50:04.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27145/vernacular_metadata_2024-02-02T23:41:49/mapped_metadata_2024-02-02T23:46:42/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27147/vernacular_metadata_2024-02-02T23:41:52/mapped_metadata_2024-02-02T23:46:44/validation_2024-02-02T23:50:06.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27147/vernacular_metadata_2024-02-02T23:41:52/mapped_metadata_2024-02-02T23:46:44/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27149/vernacular_metadata_2024-02-02T23:41:56/mapped_metadata_2024-02-02T23:46:51/validation_2024-02-02T23:50:07.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27149/vernacular_metadata_2024-02-02T23:41:56/mapped_metadata_2024-02-02T23:46:51/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27150/vernacular_metadata_2024-02-02T23:41:57/mapped_metadata_2024-02-02T23:46:52/validation_2024-02-02T23:50:07.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27150/vernacular_metadata_2024-02-02T23:41:57/mapped_metadata_2024-02-02T23:46:52/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27165/vernacular_metadata_2024-02-02T23:41:58/mapped_metadata_2024-02-02T23:46:55/validation_2024-02-02T23:50:17.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27165/vernacular_metadata_2024-02-02T23:41:58/mapped_metadata_2024-02-02T23:46:55/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27433/vernacular_metadata_2024-02-02T23:42:21/mapped_metadata_2024-02-02T23:47:40/validation_2024-02-02T23:50:19.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27433/vernacular_metadata_2024-02-02T23:42:21/mapped_metadata_2024-02-02T23:47:40/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27618/vernacular_metadata_2024-02-02T23:42:27/mapped_metadata_2024-02-02T23:47:42/validation_2024-02-02T23:50:22.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27618/vernacular_metadata_2024-02-02T23:42:27/mapped_metadata_2024-02-02T23:47:42/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27619/vernacular_metadata_2024-02-02T23:42:40/mapped_metadata_2024-02-02T23:47:47/validation_2024-02-02T23:50:22.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27619/vernacular_metadata_2024-02-02T23:42:40/mapped_metadata_2024-02-02T23:47:47/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27620/vernacular_metadata_2024-02-02T23:42:41/mapped_metadata_2024-02-02T23:47:47/validation_2024-02-02T23:50:23.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27620/vernacular_metadata_2024-02-02T23:42:41/mapped_metadata_2024-02-02T23:47:47/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27621/vernacular_metadata_2024-02-02T23:42:44/mapped_metadata_2024-02-02T23:48:01/validation_2024-02-02T23:50:25.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27621/vernacular_metadata_2024-02-02T23:42:44/mapped_metadata_2024-02-02T23:48:01/data/
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27622/vernacular_metadata_2024-02-02T23:42:57/mapped_metadata_2024-02-02T23:48:05/validation_2024-02-02T23:50:25.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27622/vernacular_metadata_2024-02-02T23:42:57/mapped_metadata_2024-02-02T23:48:05/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27623/vernacular_metadata_2024-02-02T23:42:58/mapped_metadata_2024-02-02T23:48:07/validation_2024-02-02T23:50:26.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27623/vernacular_metadata_2024-02-02T23:42:58/mapped_metadata_2024-02-02T23:48:07/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27624/vernacular_metadata_2024-02-02T23:42:59/mapped_metadata_2024-02-02T23:48:09/validation_2024-02-02T23:50:27.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27624/vernacular_metadata_2024-02-02T23:42:59/mapped_metadata_2024-02-02T23:48:09/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27625/vernacular_metadata_2024-02-02T23:43:06/mapped_metadata_2024-02-02T23:48:14/validation_2024-02-02T23:50:43.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27625/vernacular_metadata_2024-02-02T23:43:06/mapped_metadata_2024-02-02T23:48:14/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27667/vernacular_metadata_2024-02-02T23:44:25/mapped_metadata_2024-02-02T23:49:06/validation_2024-02-02T23:50:43.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27667/vernacular_metadata_2024-02-02T23:44:25/mapped_metadata_2024-02-02T23:49:06/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27668/vernacular_metadata_2024-02-02T23:44:26/mapped_metadata_2024-02-02T23:49:08/validation_2024-02-02T23:50:44.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27668/vernacular_metadata_2024-02-02T23:44:26/mapped_metadata_2024-02-02T23:49:08/data/
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27669/vernacular_metadata_2024-02-02T23:44:28/mapped_metadata_2024-02-02T23:49:10/validation_2024-02-02T23:50:44.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27669/vernacular_metadata_2024-02-02T23:44:28/mapped_metadata_2024-02-02T23:49:10/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27670/vernacular_metadata_2024-02-02T23:44:28/mapped_metadata_2024-02-02T23:49:12/validation_2024-02-02T23:50:45.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27670/vernacular_metadata_2024-02-02T23:44:28/mapped_metadata_2024-02-02T23:49:12/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27695/vernacular_metadata_2024-02-02T23:44:29/mapped_metadata_2024-02-02T23:49:13/validation_2024-02-02T23:50:46.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27695/vernacular_metadata_2024-02-02T23:44:29/mapped_metadata_2024-02-02T23:49:13/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27696/vernacular_metadata_2024-02-02T23:44:35/mapped_metadata_2024-02-02T23:49:15/validation_2024-02-02T23:50:48.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27696/vernacular_metadata_2024-02-02T23:44:35/mapped_metadata_2024-02-02T23:49:15/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27697/vernacular_metadata_2024-02-02T23:44:44/mapped_metadata_2024-02-02T23:49:18/validation_2024-02-02T23:50:56.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27697/vernacular_metadata_2024-02-02T23:44:44/mapped_metadata_2024-02-02T23:49:18/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27797/vernacular_metadata_2024-02-02T23:45:17/mapped_metadata_2024-02-02T23:49:26/validation_2024-02-02T23:50:57.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27797/vernacular_metadata_2024-02-02T23:45:17/mapped_metadata_2024-02-02T23:49:26/data/

[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27970/vernacular_metadata_2024-02-02T23:45:19/mapped_metadata_2024-02-02T23:49:27/validation_2024-02-02T23:51:02.csv
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27970/vernacular_metadata_2024-02-02T23:45:19/mapped_metadata_2024-02-02T23:49:27/data/

Error running 26725 because it is marked for ETL (while the contributor fixes the oai feed):
```
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} WARNING - ------------------------------------------------------------
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} WARNING - -------------------- Validation Errors ---------------------
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} WARNING - ------------------------------------------------------------
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} WARNING -  Collection 26725: versions.get_mapped_pages: No mapped version path provided 
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} WARNING - Traceback (most recent call last):
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} WARNING -   File "/usr/local/airflow/dags/rikolti/dags/utils_by_mapper_type.py", line 164, in validate_endpoint_task
    mapped_pages = get_mapped_pages(mapped_version)
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} WARNING -   File "/usr/local/airflow/dags/rikolti/utils/versions.py", line 160, in get_mapped_pages
    raise ValueError("versions.get_mapped_pages: No mapped version path provided")
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} WARNING - ValueError: versions.get_mapped_pages: No mapped version path provided
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} WARNING - <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - ************************************************************
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - 1 collections encountered validation errors
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - please validate manually: [26725]
[2024-02-02, 23:51:02 UTC] {{logging_mixin.py:150}} INFO - ************************************************************

```

**Comment by christinklez** [comment]
Accepted on 2024-02-02

**Comment by christinklez** [comment]
Review 26725 [oai output updated to include the is_shown_by dependent identifier]

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/26724/vernacular_metadata_2024-02-07T20:14:28/mapped_metadata_2024-02-07T20:20:36/validation_2024-02-07T20:24:57.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#26724/vernacular_metadata_2024-02-07T20:14:28/mapped_metadata_2024-02-07T20:20:36/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/26725/vernacular_metadata_2024-02-07T20:15:34/mapped_metadata_2024-02-07T20:20:56/validation_2024-02-07T20:25:00.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#26725/vernacular_metadata_2024-02-07T20:15:34/mapped_metadata_2024-02-07T20:20:56/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27121/vernacular_metadata_2024-02-07T20:15:46/mapped_metadata_2024-02-07T20:20:58/validation_2024-02-07T20:25:01.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27121/vernacular_metadata_2024-02-07T20:15:46/mapped_metadata_2024-02-07T20:20:58/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27122/vernacular_metadata_2024-02-07T20:15:48/mapped_metadata_2024-02-07T20:20:59/validation_2024-02-07T20:25:01.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27122/vernacular_metadata_2024-02-07T20:15:48/mapped_metadata_2024-02-07T20:20:59/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27140/vernacular_metadata_2024-02-07T20:15:49/mapped_metadata_2024-02-07T20:20:59/validation_2024-02-07T20:25:03.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27140/vernacular_metadata_2024-02-07T20:15:49/mapped_metadata_2024-02-07T20:20:59/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27144/vernacular_metadata_2024-02-07T20:15:52/mapped_metadata_2024-02-07T20:21:27/validation_2024-02-07T20:25:06.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27144/vernacular_metadata_2024-02-07T20:15:52/mapped_metadata_2024-02-07T20:21:27/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27145/vernacular_metadata_2024-02-07T20:15:59/mapped_metadata_2024-02-07T20:21:31/validation_2024-02-07T20:25:08.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27145/vernacular_metadata_2024-02-07T20:15:59/mapped_metadata_2024-02-07T20:21:31/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27147/vernacular_metadata_2024-02-07T20:16:03/mapped_metadata_2024-02-07T20:21:34/validation_2024-02-07T20:25:10.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27147/vernacular_metadata_2024-02-07T20:16:03/mapped_metadata_2024-02-07T20:21:34/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27149/vernacular_metadata_2024-02-07T20:16:08/mapped_metadata_2024-02-07T20:21:40/validation_2024-02-07T20:25:10.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27149/vernacular_metadata_2024-02-07T20:16:08/mapped_metadata_2024-02-07T20:21:40/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27150/vernacular_metadata_2024-02-07T20:16:08/mapped_metadata_2024-02-07T20:21:41/validation_2024-02-07T20:25:11.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27150/vernacular_metadata_2024-02-07T20:16:08/mapped_metadata_2024-02-07T20:21:41/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27165/vernacular_metadata_2024-02-07T20:16:09/mapped_metadata_2024-02-07T20:21:45/validation_2024-02-07T20:25:21.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27165/vernacular_metadata_2024-02-07T20:16:09/mapped_metadata_2024-02-07T20:21:45/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27433/vernacular_metadata_2024-02-07T20:16:34/mapped_metadata_2024-02-07T20:22:30/validation_2024-02-07T20:25:23.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27433/vernacular_metadata_2024-02-07T20:16:34/mapped_metadata_2024-02-07T20:22:30/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27618/vernacular_metadata_2024-02-07T20:16:41/mapped_metadata_2024-02-07T20:22:32/validation_2024-02-07T20:25:26.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27618/vernacular_metadata_2024-02-07T20:16:41/mapped_metadata_2024-02-07T20:22:32/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27619/vernacular_metadata_2024-02-07T20:16:57/mapped_metadata_2024-02-07T20:22:36/validation_2024-02-07T20:25:27.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27619/vernacular_metadata_2024-02-07T20:16:57/mapped_metadata_2024-02-07T20:22:36/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27620/vernacular_metadata_2024-02-07T20:16:59/mapped_metadata_2024-02-07T20:22:36/validation_2024-02-07T20:25:27.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27620/vernacular_metadata_2024-02-07T20:16:59/mapped_metadata_2024-02-07T20:22:36/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27621/vernacular_metadata_2024-02-07T20:17:02/mapped_metadata_2024-02-07T20:22:50/validation_2024-02-07T20:25:30.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27621/vernacular_metadata_2024-02-07T20:17:02/mapped_metadata_2024-02-07T20:22:50/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27622/vernacular_metadata_2024-02-07T20:17:17/mapped_metadata_2024-02-07T20:22:55/validation_2024-02-07T20:25:31.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27622/vernacular_metadata_2024-02-07T20:17:17/mapped_metadata_2024-02-07T20:22:55/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27623/vernacular_metadata_2024-02-07T20:17:18/mapped_metadata_2024-02-07T20:22:57/validation_2024-02-07T20:25:31.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27623/vernacular_metadata_2024-02-07T20:17:18/mapped_metadata_2024-02-07T20:22:57/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27624/vernacular_metadata_2024-02-07T20:17:19/mapped_metadata_2024-02-07T20:23:00/validation_2024-02-07T20:25:33.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27624/vernacular_metadata_2024-02-07T20:17:19/mapped_metadata_2024-02-07T20:23:00/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27625/vernacular_metadata_2024-02-07T20:17:28/mapped_metadata_2024-02-07T20:23:05/validation_2024-02-07T20:25:50.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27625/vernacular_metadata_2024-02-07T20:17:28/mapped_metadata_2024-02-07T20:23:05/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27667/vernacular_metadata_2024-02-07T20:18:53/mapped_metadata_2024-02-07T20:24:01/validation_2024-02-07T20:25:50.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27667/vernacular_metadata_2024-02-07T20:18:53/mapped_metadata_2024-02-07T20:24:01/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27668/vernacular_metadata_2024-02-07T20:18:54/mapped_metadata_2024-02-07T20:24:04/validation_2024-02-07T20:25:51.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27668/vernacular_metadata_2024-02-07T20:18:54/mapped_metadata_2024-02-07T20:24:04/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27669/vernacular_metadata_2024-02-07T20:18:55/mapped_metadata_2024-02-07T20:24:06/validation_2024-02-07T20:25:51.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27669/vernacular_metadata_2024-02-07T20:18:55/mapped_metadata_2024-02-07T20:24:06/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27670/vernacular_metadata_2024-02-07T20:18:57/mapped_metadata_2024-02-07T20:24:09/validation_2024-02-07T20:25:51.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27670/vernacular_metadata_2024-02-07T20:18:57/mapped_metadata_2024-02-07T20:24:09/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27695/vernacular_metadata_2024-02-07T20:18:57/mapped_metadata_2024-02-07T20:24:10/validation_2024-02-07T20:25:53.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27695/vernacular_metadata_2024-02-07T20:18:57/mapped_metadata_2024-02-07T20:24:10/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27696/vernacular_metadata_2024-02-07T20:19:07/mapped_metadata_2024-02-07T20:24:12/validation_2024-02-07T20:25:55.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27696/vernacular_metadata_2024-02-07T20:19:07/mapped_metadata_2024-02-07T20:24:12/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27697/vernacular_metadata_2024-02-07T20:19:18/mapped_metadata_2024-02-07T20:24:15/validation_2024-02-07T20:26:03.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27697/vernacular_metadata_2024-02-07T20:19:18/mapped_metadata_2024-02-07T20:24:15/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27797/vernacular_metadata_2024-02-07T20:20:00/mapped_metadata_2024-02-07T20:24:24/validation_2024-02-07T20:26:04.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27797/vernacular_metadata_2024-02-07T20:20:00/mapped_metadata_2024-02-07T20:24:24/data/

[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27970/vernacular_metadata_2024-02-07T20:20:03/mapped_metadata_2024-02-07T20:24:25/validation_2024-02-07T20:26:10.csv
[2024-02-07, 20:26:10 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27970/vernacular_metadata_2024-02-07T20:20:03/mapped_metadata_2024-02-07T20:24:25/data/
